frontend/src/config/github-enterprise.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * GitHub Enterprise Server configuration
+ */
+
+/**
+ * Initialize GitHub Enterprise Server configuration from environment variables
+ */
+export const initGitHubEnterpriseConfig = () => {
+  // Set GitHub Enterprise URL if available
+  if (import.meta.env.VITE_GITHUB_ENTERPRISE_URL) {
+    window.GITHUB_ENTERPRISE_URL = import.meta.env.VITE_GITHUB_ENTERPRISE_URL;
```

Comment:
Could we avoid using `window` to pass around GitHub related settings. We'd rather just use the `UserSettings` because its persists for local installations as well